datastore/package.json

```diff
@@ -4,6 +4,9 @@
   "version": "0.0.1",
   "private": true,
   "license": "Apache Version 2.0",
+  "bin": {
```

Comment:
All of the samples in the repo should be consistent on how they instruct the user to run the sample. Do you think `npm link` and `"bin"` is better than how the `storage/` samples does it? I'm leaning towards how the `storage/` sample does it because it keeps everything local to the sample without affecting something elsewhere on the user's comp. What are your thoughts?